lib/owlmoji.js

```diff
@@ -1,16 +1,51 @@
-const { nameToEmoji } = require('gemoji');
+const { gemoji, nameToEmoji } = require('gemoji');
 
-const owlmoji = ['owlbert-books', 'owlbert-mask', 'owlbert', 'owlbert-reading', 'owlbert-thinking'];
+const owlmoji = [
+  {
+    names: ['owlbert'],
+    tags: ['owlbert'],
+    description: 'owlbert',
+    category: 'ReadMe',
+  },
+  {
+    names: ['owlbert-books'],
+    tags: ['owlbert'],
+    description: 'owlbert carrying books',
+    category: 'ReadMe',
+  },
+  {
+    names: ['owlbert-mask'],
+    tags: ['owlbert with a respirator'],
+    description: 'owlbert',
+    category: 'ReadMe',
+  },
+  {
+    names: ['owlbert-reading'],
+    tags: ['owlbert reading'],
+    description: 'owlbert',
+    category: 'ReadMe',
+  },
+  {
+    names: ['owlbert-thinking'],
+    tags: ['owlbert thinking'],
+    description: 'owlbert',
+    category: 'ReadMe',
+  },
+];
```

Comment:
the `tag`s and `description`s here seem a little inconsistent

Reply:
Oops. It also doesn't really matter, nobody is consuming those.

Reply:
Good catch, fixed it up.